solutions/2400-2499/2413-smallest-even-multiple-easy.md

```diff
@@ -34,16 +34,16 @@ Explanation: The smallest multiple of both 6 and 2 is 6. Note that a number is a
 
 ## Approach 1: If-else statement
 
-<Tabs>
-<TabItem value="java" label="Java">
-<SolutionAuthor name="@kat-kan"/>
-
 The smallest multiple of 2 and a number will be either:
 - the number if it is even or
 - the number multiplied by 2.
 
 In solution we need to first check if the number is even and return the correct result.
 
+<Tabs>
+<TabItem value="java" label="Java">
+<SolutionAuthor name="@kat-kan"/>
+
 ```java
 class Solution {
     public int smallestEvenMultiple(int n) {
```

Comment:
This solution can be more clean. `result` is not necessary at all.

````
class Solution {
    public int smallestEvenMultiple(int n) {
        if (n % 2 == 0) return n;
        return n * 2;
    }
}
````

Reply:
fixed